numpy/core/numeric.py

```diff
@@ -6,7 +6,22 @@
 import sys
 import warnings
 
-from numpy.core import multiarray
+from . import multiarray
+from .multiarray import (CLIP, WRAP, RAISE, MAXDIMS, ALLOW_THREADS,
+                         BUFSIZE, MAY_SHARE_BOUNDS, MAY_SHARE_EXACT,
+                         ndarray, flatiter, nditer, nested_iters,
+                         broadcast, dtype, copyto, arange, array, zeros,
+                         count_nonzero, empty, empty_like, fromstring,
+                         fromiter, fromfile, frombuffer, shares_memory,
+                         may_share_memory, int_asbuffer, where,
+                         concatenate, set_numeric_ops, can_cast,
+                         promote_types, min_scalar_type, result_type,
+                         lexsort, compare_chararrays, putmask, einsum,
+                         dot, inner, vdot, matmul)
+from .multiarray import _fastCopyAndTranspose as fastCopyAndTranspose
```

Comment:
Note that this could be included in the previous import
```
..., _fastCopyAndTranspose as fastCopyAndTranspose, etc.
```
I don't have a preference one way or the other for that